cmd/root.go

```diff
@@ -384,8 +391,8 @@ func overrideCluster(envName string, clientConfig clientcmd.ClientConfig, overri
 // the user passes a list of files, we will expand all templates in those files,
 // while if a user passes an environment name, we will expand all component
 // files using that environment.
-func expandEnvCmdObjs(cmd *cobra.Command, env string, components []string, cwd metadata.AbsPath) ([]*unstructured.Unstructured, error) {
-	expander, err := newExpander(cmd)
+func expandEnvCmdObjs(fs afero.Fs, cmd *cobra.Command, env string, components []string, cwd metadata.AbsPath) ([]*unstructured.Unstructured, error) {
```

Comment:
Instead of passing `fs` from every command, I think the below can just be `expander, err := newExpander(appFs, cmd)`. This should simplify this change a lot and remove most of the `fs` param passing.

Reply:
The reason I did this is so we aren't reliant on global state. I think I'll take another stab at rewriting this so the process can rely on global state as a default, but still allow you to pass a fs in as an override.